governance/policies/archiving.md

```diff
@@ -88,6 +88,7 @@ When it has been decided that a set of package versions (aka "versions") should
     - Meaning:
         - Expresses the declared intent of the maintainers to maintain only certain versions ranges of a package.
         - The value of the `x-maintenance-intent` on the latest published package will precedence.
+    - Default value: ["(any)"] on 2025-01-17, please note that this default may change
     - Examples:
         - `["(latest)"]` the maintainer will only maintain the latest version
```

Comment:
```suggestion
        - `["(latest)"]` the maintainer will only maintain the latest version[^caveat]
        [^caveat]: note that this will retain the latest versions of this package so that every supported OCaml version will have an installation candidate.
```
I think we should make this semantic explicit